### ISSUE TITLE
Make submission workflow explicit

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#3357]
# The problem

We spend a lot of time directing people in the helpdesk how to get to basic/needed functions.

"First click here, then click on... well, click on whatever words are linked in that table, then in the upper right, click on 'Manage Access,' then..."

This is, in no small part, because we're using an inherited design that does not acknowledge the experiential reality for many of our users coming to a new system, often for the first time, and possibly only once per year.

We currently redirect **all** users to a page that looks like this:

![image](https://github.com/GSA-TTS/FAC/assets/38290/6b93f323-8580-4eed-b6fc-b27c02778ccf)

This page looks OK, because we (the developers of the FAC) only ever have one or two audits "live" at a time. Auditors working as part of a team can end up with *dozens* if not more. Add in the complexity of having multiple audits that are incomplete (and incorrect), and no way to remove those audits, and we now have a recipe for confusion and poor user experience.

From a heuristic perspective, we have further problems:

1. To get to an audit, we have to click a link that has constantly changing text. Should I click "In Progress" to get to the audit, or "Auditor Certified?"
2. Is the link for "Auditor Certified" the same, or different, from "Unlock?" 
3. Which audits are current, and which are old? How is this page sorted? Are my "last edited" audits at the top? Are the stale/bad audits at the bottom?
4. Why can I unlock from this page, but not manage access?

This is further problematic for Federal users, because a Federal user will **never** submit an audit. Yet, this is where they go.

## Making choices obvious

**In order to** Do what I need to do
**as a** user
**I want** to have actions available to me to be clear and obvious.

We should redirect users to a page that looks more like this:

![image](https://github.com/GSA-TTS/FAC/assets/38290/c84127d9-d069-4b1f-a6be-b2f114349856)

This makes absolutely clear what actions a user can take.

Eventually, we can shape this to reflect the user role. For example, Federal users may end up with a very different set of actions. For now, this would at least make "Search" an obvious action that a Federal user could take. (Or, authenticated Federal users *could just be redirected to the search page...*)

# Acceptance Criteria

### Scenario: I am an authenticated Federal user

**Given**   that I will never be *submitting* an audit
**when**  I log in
...

[comment]: # "Each task should be a verifiable outcome"
```[tasklist]
### then... 
- [ ] I should be directed to the search page.
```

### Scenario: I am an authenticated public user

**Given**   that I might search or submit an audit
**when**  I log in
...

[comment]: # "Each task should be a verifiable outcome"
```[tasklist]
### then... 
- [ ] I should be directed to a submission dashboard.
```

## Tasks

```[tasklist]
### Tasks
- [ ] Design page
- [ ] Ask three auditors/auditees if they would like this better
- [ ] Do it.
``` 

### Shepherd

[comment]: # "@ mention shepherds as we move across the board."

* Design shepherd: 
* Engineering shepherd: 

# Background

[comment]: # "Any helpful contextual notes or links to artifacts/evidence, if needed"

# Security Considerations

Required per [CM-4](https://nvd.nist.gov/800-53/Rev4/control/CM-4).

[comment]: # "Our SSP says 'The team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.'"
[comment]: # "Please do not remove this section without care."
[comment]: # "Note any security concerns that might be implicated in the change. 'None' is OK, but we must be explicit here."

# Sketch

We're thinking we'll try implementing A and then changing B.

```[tasklist]
# Tasks
- [ ] [something we'll definitely have to do]
```

---

<details>
  <summary>Process checklist</summary>
  
# Sketch

[comment]: # "Notes or a checklist reflecting our understanding of the selected approach"

- [ ] Design designs all the things
- [ ] Engineering engineers all the things

# Definition of Done

### If not likely to be important in the next quarter...
- [ ] Archived from the board

### Otherwise...

- [ ] Has a clear story statement
- [ ] Design or Engineering accepts that it belongs in their respective backlog

## Design Backlog

- [ ] Has clearly stated/testable acceptance criteria
- [ ] Meets the design Definition of Ready [citation needed]
- [ ] A design shepherd has been identified

## Design In Progress

- [ ] Meets the design Definition of Done [citation needed]

## Design Review Needed

- [ ] Necessary outside review/sign-off was provided

## Design Done

- [ ] Presented in a sprint review
- [ ] Includes screenshots or references to artifacts

### If no engineering is necessary
- [ ] Tagged with the sprint where it was finished
- [ ] Archived

## Engineering Backlog

- [ ] Has clearly stated/testable acceptance criteria
- [ ] Has a sketch or list of tasks
- [ ] Can reasonably be done in a few days (otherwise, split this up!)

## Engineering Available

- [ ] There's capacity in the `In Progress` column
- [ ] An engineering shepherd has been identified

## Engineering In Progress

- [ ] Meets acceptance criteria
- [ ] Meets [QASP conditions](https://derisking-guide.18f.gov/qasp/)

### If there's UI...
- [ ] Screen reader - Listen to the experience with a screen reader extension, ensure the information presented in order
- [ ] Keyboard navigation - Run through acceptance criteria with keyboard tabs, ensure it works. 
- [ ] Text scaling - Adjust viewport to 1280 pixels wide and zoom to 200%, ensure everything renders as expected. Document 400% zoom issues with USWDS if appropriate.

## Engineering Blocked

- [ ] Blocker removed/resolved

## Engineering Review Needed

- [ ] Outside review/sign-off was provided

## Engineering Done

- [ ] Presented in a sprint review
- [ ] Includes screenshots or references to artifacts
- [ ] Tagged with the sprint where it was finished
- [ ] Archived

</details>
